library/src/main/java/com/leinardi/android/speeddial/UiUtils.java

```diff
@@ -155,8 +158,9 @@ public void run() {
     public static void rotateForward(View view, boolean animate) {
         ViewCompat.animate(view)
                 .rotation(ROTATION_ANGLE)
-                .withLayer()
-                .setDuration(animate ? SHORT_ANIM_TIME : 0)
+                .withLayer
+                        ()
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.whitespace.MethodParamPadCheck
**Severity**: ERROR
**File**: library/src/main/java/com/leinardi/android/speeddial/UiUtils.java L162

'(' should be on the previous line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<325708204>*